src/caculate.test.js

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import Calculator from './components/Calculator';
```

Comment:
- [ ] You added a snapshot test for only the Calculator component, but you should add the snapshot for all components. Kindly do the same for the Navbar, Home and Quote components.

Reply:
Got it, @julie-ify.